src/platform/with_wgl/native_gl_context.rs

```diff
@@ -54,10 +54,13 @@ impl Drop for NativeGLContext {
     fn drop(&mut self) {
         unsafe {
             if !self.weak {
+                self.unbind().unwrap(); //wglDeleteContext requires unbind
                 wgl::DeleteContext(self.render_ctx as *const _);
                 let window = user32::WindowFromDC(self.device_ctx);
-                user32::ReleaseDC(window, self.device_ctx);
-                user32::DestroyWindow(window);
+                if !window.is_null() {
```

Comment:
We are not creating multiple contexts associated with the same HDC. It was just a extra safety check because null is a possible return value of WindowFromDC. It shouldn't happen in our code, but it seems safer to check for null.